SayTheirNames/Source/View/Hashtag/HashtagView.swift

```diff
@@ -39,10 +39,13 @@ class HashtagView: UIView {
         return label
     }()
 
+    private var originalText = "#JUSTICEFORFIRST"
```

Comment:
If this never changes lets make it a constant.

Reply:
It does change- on line 84, when the user taps on a hashtag, it gets changed to the actual text of the hashtag so it can be put back when they let go